sdl_viewer/src/lib.rs

```diff
@@ -271,6 +277,41 @@ impl PointCloudRenderer {
     }
 }
 
+#[derive(Debug,Serialize,Deserialize)]
+pub struct CameraStates {
+    states: Vec<camera::State>,
+}
+
+fn save_camera(index: usize, pose_path: &Option<PathBuf>, camera: &Camera) {
+    if pose_path.is_none() {
+        println!("Not serving from a local directory. Cannot save camera.");
+        return;
+    }
+    assert!(index < 10);
+    let mut states = ::std::fs::read_to_string(pose_path.as_ref().unwrap()).and_then(|data| {
```

Comment:
Nit: `use std::fs;`, as you do for cmp, io, thread, etc. and then just `fs::read_to_string` here.

Reply:
Will do in the followup.